sympy/matrices/tests/test_matrices.py

```diff
@@ -480,12 +481,19 @@ def test_expand():
 def test_random():
     M = randMatrix(3, 3)
     M = randMatrix(3, 3, seed=3)
+    assert M == randMatrix(3, 3, seed=3)
+
     M = randMatrix(3, 4, 0, 150)
-    M = randMatrix(3, symmetric=True)
+    M = randMatrix(3, seed=4, symmetric=True)
+    assert M == randMatrix(3, seed=4, symmetric=True)
+
     S = M.copy()
     S.simplify()
     assert S == M  # doesn't fail when elements are Numbers, not int
 
+    rng = random.Random(4)
+    assert M == randMatrix(3, prng=rng, symmetric=True)
```

Comment:
You can use `prng` instead of `rng` here.
Or just write `assert M == randMatrix(3, prng=random.Random(4), symmetric=True)`.